internal/resources/oncall/resource_outgoing_webhook.go

```diff
@@ -29,70 +65,86 @@ func resourceOutgoingWebhook() *common.Resource {
+				Description:      "The webhook URL. Required when not using a preset that controls this field.",
+				DiffSuppressFunc: suppressDiffForPresetControlledField("url"),
 			},
 			"data": {
-				Type:        schema.TypeString,
-				Optional:    true,
-				Description: "The data of the webhook.",
+				Type:             schema.TypeString,
+				Optional:         true,
+				Description:      "The data of the webhook.",
+				DiffSuppressFunc: suppressDiffForPresetControlledField("data"),
 			},
 			"user": {
-				Type:        schema.TypeString,
-				Optional:    true,
-				Description: "Username to use when making the outgoing webhook request.",
+				Type:             schema.TypeString,
+				Optional:         true,
+				Description:      "Username to use when making the outgoing webhook request.",
+				DiffSuppressFunc: suppressDiffForPresetControlledField("user"),
 			},
 			"password": {
-				Type:        schema.TypeString,
-				Optional:    true,
-				Description: "The auth data of the webhook. Used for Basic authentication",
-				Sensitive:   true,
+				Type:             schema.TypeString,
+				Optional:         true,
+				Description:      "The auth data of the webhook. Used for Basic authentication",
+				Sensitive:        true,
+				DiffSuppressFunc: suppressDiffForPresetControlledField("password"),
 			},
 			"authorization_header": {
-				Type:        schema.TypeString,
-				Optional:    true,
-				Description: "The auth data of the webhook. Used in Authorization header instead of user/password auth.",
-				Sensitive:   true,
+				Type:             schema.TypeString,
+				Optional:         true,
+				Description:      "The auth data of the webhook. Used in Authorization header instead of user/password auth.",
+				Sensitive:        true,
+				DiffSuppressFunc: suppressDiffForPresetControlledField("authorization_header"),
 			},
 			"forward_whole_payload": {
-				Type:        schema.TypeBool,
-				Optional:    true,
-				Description: "Toggle to send the entire webhook payload instead of using the values in the Data field.",
+				Type:             schema.TypeBool,
+				Optional:         true,
+				Description:      "Toggle to send the entire webhook payload instead of using the values in the Data field.",
+				DiffSuppressFunc: suppressDiffForPresetControlledField("forward_whole_payload"),
 			},
 			"trigger_type": {
-				Type:        schema.TypeString,
-				Optional:    true,
-				Description: "The type of event that will cause this outgoing webhook to execute. The types of triggers are: `escalation`, `alert group created`, `acknowledge`, `resolve`, `silence`, `unsilence`, `unresolve`, `unacknowledge`.",
-				Default:     "escalation",
+				Type:             schema.TypeString,
+				Optional:         true,
+				Description:      "The type of event that will cause this outgoing webhook to execute. The types of triggers are: `escalation`, `alert group created`, `acknowledge`, `resolve`, `silence`, `unsilence`, `unresolve`, `unacknowledge`.",
```

Comment:
Should we list the other trigger types here and group them to the relevant preset?